core/arch/arm/mm/tee_pager.c

```diff
@@ -865,7 +865,6 @@ static void split_region(struct vm_paged_region *reg,
 
 	pager_unlock(exceptions);
 }
-DECLARE_KEEP_PAGER(split_region);
```

Comment:
`DECLARE_KEEP_PAGER()` is used here to make sure that we don't get pagefault for this code while holing the pager lock, same below.

Reply:
Ok, i'll have a second look. Thanks.